bootstrap.php

```diff
@@ -650,6 +652,7 @@ function ($app) {
         $commandBus->subscribe($app['place_geocoordinates_command_handler']);
         $commandBus->subscribe($app['event_geocoordinates_command_handler']);
         $commandBus->subscribe($app['organizer_geocoordinates_command_handler']);
+        $commandBus->subscribe($app[ProductionCommandHandler::class]);
```

Comment:
Good thinking of using class constants as service definitions already 👍 
It's something we discussed multiple times in the past I think, but it's easy to forget once you're actually working in these files with string constants.